content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
+**React പഠിക്കുന്നതിനും** **ഒരു [single-page](/docs/glossary.html#single-page-application) React App** മികച്ച രീതിയിൽ ഉണ്ടാക്കിയെടുക്കുന്നതിനുള്ള ഏറ്റവും സുഖകരമായ സാഹചര്യം സൃഷ്ടിക്കുന്ന toolchain ആണ് [Create React App](https://github.com/facebookincubator/create-react-app).
 
-It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have Node >= 8.10 and npm >= 5.6 on your machine. To create a project, run:
+ഇത് നിങ്ങൾക്ക് വേണ്ടി ഏറ്റവും പുതിയ JavaScript ഫീച്ചറുകൾ അടങ്ങിയ  മികച്ച ഒരു ഡെവലപ്പ്മെന്റ് സാഹചര്യം ഒരുക്കിത്തരികയും പ്രൊഡക്ഷനു വേണ്ടി നിങ്ങളുടെ app-നെ മികച്ചതാക്കി മാറ്റിയെടുക്കുകയും ചെയ്യുന്നു. ഇതിനായി നിങ്ങളുടെ കമ്പ്യൂട്ടറിൽ Node-ന്റെ 8.10-നേക്കാൾ ഉയർന്ന പതിപ്പും npm-ന്റെ 5.6-നേക്കാൾ ഉയർന്ന പതിപ്പും ഉണ്ടായിരിക്കണം. ഒരു പ്രൊജക്റ്റ് നിർമ്മിക്കുന്നതിനായി താഴെയുള്ള കമാൻഡ് റൺ ചെയ്യുക:
 
 ```bash
 npx create-react-app my-app
 cd my-app
 npm start
 ```
 
->Note
+>കുറിപ്പ്
 >
->`npx` on the first line is not a typo -- it's a [package runner tool that comes with npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
+>`npx` എന്നത് ടൈപ്പ് ചെയ്തപ്പോൾ തെറ്റിയതല്ല -- അത് [npm 5.2-നു മുകളിൽ ഉള്ള പതിപ്പുകളോടൊപ്പം വരുന്ന ഒരു പാക്കേജ് റണ്ണർ ടൂൾ](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) ആണ്.
 
-Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](https://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+Create React App ഒരിക്കലും backend ലോജിക്കുകളോ ഡാറ്റാബേസുകളോ കൈകാര്യം ചെയ്യില്ല; ഇത് ഒരു frontend build pipeline മാത്രം ഉണ്ടാക്കുന്നതിനാൽ നിങ്ങൾക്ക് ഇഷ്ടമുള്ള backend-നോടോപ്പം ഉപയോഗിക്കാവുന്നതാണ്. അടിസ്ഥാനപരമായി ഇത് ഉപയോഗിച്ചിരിക്കുന്നത് [Babel](https://babeljs.io/)-ഉം [webpack](https://webpack.js.org/)-ഉം ആണ്. എന്നാൽ നിങ്ങൾ അതിനെ കുറിച്ചെല്ലാം അറിഞ്ഞിരിക്കണം എന്ന് നിർബന്ധമില്ല.
 
-When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app--) and the [User Guide](https://facebook.github.io/create-react-app/).
+നിങ്ങൾ പ്രൊഡക്ഷനിലേക്ക് ഡിപ്ലോയ് ചെയ്യാൻ തയ്യാറാകുമ്പോൾ `npm run build` എന്ന കമാൻഡ് റൺ ചെയ്‌താൽ `build` ഫോൾഡറിനകത്ത് നിങ്ങളുടെ app-ന്റെ ഏറ്റവും മികവാർന്ന ഒരു ബിൽഡ് ഉണ്ടായി വരുന്നതാണ്. നിങ്ങൾക്ക് Create React App-നെ കുറിച്ച് കൂടുതൽ മനസ്സിലാക്കാൻ [അതിന്റെ README](https://github.com/facebookincubator/create-react-app#create-react-app--) യോ [യൂസർ ഗൈഡോ](https://facebook.github.io/create-react-app/) നോക്കാവുന്നതാണ്.
```

Comment:
Please leave the word 'optimized' as it is because I think the meaning is not completely conveyed in the current choice of word.